### ISSUE TITLE
[feat] better ONNX support

### ISSUE THREAD

**Issue by shijianjian** [kornia/kornia#1153]
## 🚀 Feature
Whilst testing on exporting kornia modules into ONNX, there are several operators that are currently not supported by pytorch.onnx. Specifically:
- ```torch.linalg.solve```
- ```torch.inverse```

For a better support of ONNX/TensorRT export, we might need to fix this.

**Comment by edgarriba** [comment]
Maybe similar to JIT, and module, we should test export to onnx

**Comment by shijianjian** [comment]
Indeed. I have talked with mmlab, they seem to be interested in exporting image processing ops as ONNX, TensorRT, etc. We need to somehow take the image proc export seriously. I will start from providing [kornia docker](https://github.com/shijianjian/Kornia-ONNX-docker). In some point, I think we gonna push one or two Docker containers to DockerHub to help people easy export.

**Comment by edgarriba** [comment]
@shijianjian we could create a docker hub account and maintain images there. @Borda any idea here ?

**Comment by shijianjian** [comment]
Having a docker hub account is probably the best approach to go since we might have many cpp code to support the conversion if it is needed and we cannot find workarounds from PyTorch team. There are several operations that are not supported like ```affine_grid_generator```.
